landlab/components/flow_accum/lossy_flow_accumulator.py

```diff
@@ -0,0 +1,464 @@
+
+    `run_one_step` takes the optional argument update_flow_director (default is
+    True) that determines if the flow_director is re-run before flow is
+    accumulated.
+
+    Parameters
+    ----------
+    grid : ModelGrid
+        A Landlab grid.
+    surface : field name at node or array of length node
+        The surface to direct flow across.
+    flow_director : string, class, instance of class.
+        A string of method or class name (e.g. 'D8' or 'FlowDirectorD8'), an
+        uninstantiated FlowDirector class, or an instance of a FlowDirector
+        class. This sets the method used to calculate flow directions.
+        Default is 'FlowDirectorSteepest'
+    runoff_rate : field name, array, or float, optional (m/time)
+        If provided, sets the runoff rate and will be assigned to the grid
+        field 'water__unit_flux_in'. If a spatially and and temporally variable
+        runoff rate is desired, pass this field name and update the field
+        through model run time. If both the field and argument are present at
+        the time of initialization, runoff_rate will *overwrite* the field. If
+        neither are set, defaults to spatially constant unit input.
+    depression_finder : string, class, instance of class, optional
+        A string of class name (e.g., 'DepressionFinderAndRouter'), an
+        uninstantiated DepressionFinder class, or an instance of a
+        DepressionFinder class.
+        This sets the method for depression finding.
+    loss_function : Python function, optional
+        A function of the form f(Qw, [node_ID, [linkID, [grid]]]), where Qw is
+        the discharge at a node, node_ID the ID of the node at which the loss
+        is to be calculated, linkID is the ID of the link down which the
+        outflow drains, and grid is a Landlab ModelGrid. Note that if a linkID
+        is needed, a nodeID must also be specified, even if only as a dummy
+        parameter; similarly, if a grid is to be passed, all of the preceding
+        parameters must be specified. Both nodeID and linkID are required to
+        permit spatially variable losses, and also losses dependent on flow
+        path geometry (e.g., flow length). The grid is passed to allow fields
+        or grid properties describing values across the grid to be accessed
+        for the loss calculation (see examples).
+        This function should take (float, [int, [int, [ModelGrid]]]), and
+        return a single float.
+    **kwargs : optional
+        Any additional parameters to pass to a FlowDirector or
+        DepressionFinderAndRouter instance (e.g., partion_method for
+        FlowDirectorMFD). This will have no effect if an instantiated
+        component is passed using the flow_director or depression_finder
+        keywords.
+
+    Examples
+    --------
+    These examples pertain only to the LossyFlowAccumulator. See the main
+    FlowAccumulator documentation for more generic and comprehensive examples.
+
```

Comment:
Add a link to the FlowAccumulator documentation.

Reply:
Remind me how to do this from a docstring? Not sure if I ever figured that out... Is it just conventional markup?